go-controller/pkg/ovn/policy.go

```diff
@@ -838,9 +839,11 @@ func (oc *DefaultNetworkController) handleLocalPodSelectorAddFunc(np *networkPol
 		var err error
 		// add pods to policy port group
 		var ops []ovsdb.Operation
-		ops, err = libovsdbops.AddPortsToPortGroupOps(oc.nbClient, nil, np.portGroupName, policyPortUUIDs...)
-		if err != nil {
-			return fmt.Errorf("unable to get ops to add new pod to policy port group: %v", err)
+		if !libovsdbops.PortGroupHasPorts(oc.nbClient, np.portGroupName, policyPortUUIDs) {
+			ops, err = libovsdbops.AddPortsToPortGroupOps(oc.nbClient, nil, np.portGroupName, policyPortUUIDs...)
```

Comment:
`AddPortsToPortGroupOps` does `PortGroupHasPorts` itself, so no need to check here?